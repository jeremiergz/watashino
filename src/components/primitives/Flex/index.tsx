import React from 'react';
import Box, { BoxProps } from '../Box';

type FlexProps = BoxProps;

const Flex = ({ children, ...rest }: BoxProps) => {
    return (
        <Box display="flex" {...rest}>
            {children}
        </Box>
    );
};

Flex.displayName = 'Flex';

export { FlexProps };
export default Flex;
