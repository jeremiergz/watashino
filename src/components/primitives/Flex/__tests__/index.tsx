import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import Flex from '..';

describe('components/primitives/Flex Test Suite', () => {
    it('matches snapshot', () => {
        const jsx = <Flex>Flex Component</Flex>;
        expect(ReactTestRenderer.create(jsx).toJSON()).toMatchSnapshot();
    });
});
