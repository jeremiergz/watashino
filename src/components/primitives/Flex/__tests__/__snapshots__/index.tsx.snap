// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/primitives/Flex Test Suite matches snapshot 1`] = `
<div
  className="sc-bdVaJa cLjUOW"
  display="flex"
>
  Flex Component
</div>
`;
